packages/reactive_ble_platform_interface/lib/src/model/bonding_mode.dart

```diff
@@ -0,0 +1,13 @@
+/// Specify the bonding mode for the device.
+enum BondingMode {
```

Comment:
Like discussed above I think if we have a separate method for establish bonding we do not need this.